ext/openssl/ossl_pkey.c

```diff
@@ -83,23 +83,69 @@ ossl_pkey_new(EVP_PKEY *pkey)
 # include <openssl/decoder.h>
 
 EVP_PKEY *
-ossl_pkey_read_generic(BIO *bio, VALUE pass)
+ossl_pkey_read(BIO *bio, const char *input_type, int selection, VALUE pass)
```

Comment:
This function can be made `static`.

Reply:
Okay, I will add the `static` to the `ossl_pkey_read`. I don't understand why the `ossl_pkey_read_generic` is not static function, but the `ossl_pkey_read` is static function, seeing the following comments.

https://github.com/ruby/openssl/blob/74a41982b1dee03d6773f2adcb0e75526c1310d4/ext/openssl/ossl_pkey.h#L29-L35

Reply:
I added the `static` to the `ossl_pkey_read`.

Reply:
Ah, all right. Thanks for the explanation.